react/HOWTO.md

```diff
@@ -0,0 +1,29 @@
+# How to integrate Cozy UI to your application
+## Add Cozy-UI as a dependency
+
+Use [`yarn`](https://yarnpkg.com/) to add Cozy UI to a dependency to your project.
+
+```bash
+$ yarn add cozy-ui
+```
+
+You're now ready to use [Cozy UI's (p)React components](https://cozy.github.io/cozy-ui/react/)
+
+## Hack Cozy UI
```

Comment:
Hacking Cozy UI is not part of "How to integrate Cozy UI to your application" so I would change the titles to reflect this.